electrophysiology/models.py

```diff
@@ -28,9 +28,11 @@ class ExtracellularRecording(BaseExperimentalData):
     filter_info = models.CharField(max_length=255, null=True, blank=True,
                                    help_text="Details of hardware corner frequencies, filter type, order. TODO: make this more structured?")
     start_time = models.FloatField(null=True, blank=True,
-                                   help_text="in seconds relative to experiment start. TODO: not DateTimeField? / TimeDifference")
+                                   help_text="in seconds relative to experiment start.") 
+#								   TODO: not DateTimeField? / TimeDifference")
+# the idea is that the experiment has a single DateTime when it started, then all times are in seconds relative to this. TimeDifference could work also, but why not just float?
```

Comment:
I'm not sure why this is better than storing everything as DateTimes? The advantage of DateTimes is you can filter "all data collected on this day" very easily (or, say, "everything collected with this [faulty] appliance between the 2nd and 4th of June"): you can't do this easily when you only store offsets, since the offsets - by themselves - are completely meaningless, and you've got to perform a calculation on every row to get its *actual* time.